### ISSUE TITLE
Kendo Grid - when using Ivy, dynamic columns are rendered as auto-generated columns instead

### ISSUE THREAD

**Issue by kunajs** [telerik/kendo-angular#2762]
**Describe the bug**
Having a kendo-grid with dynamically generated columns - either normal column or spanned column (and thats why I have to use *ngIf within *ngFor ng-container) like this:
```
<kendo-grid [kendoGridBinding]="gridData" [filterable]="true" scrollable="none">
  <ng-container *ngFor="let item of columns">
    <kendo-grid-column
      *ngIf="!item.isSpanColumn"
      field="{{ item.field }}"
      title="{{ item.title }}"
      format="{{ item.format }}"
      filter="{{ item.type }}"
    ></kendo-grid-column>
    <kendo-grid-span-column *ngIf="item.isSpanColumn" title="{{ item.title }}">
      <kendo-grid-column
        *ngFor="let childColumn of item.children"
        field="{{ childColumn.field }}"
        title="{{ childColumn.title }}"
        format="{{ childColumn.format }}"
        filter="{{ childColumn.type }}"
      ></kendo-grid-column>
    </kendo-grid-span-column>
  </ng-container>
</kendo-grid>
```
Problem is when using angularCompilerOptions.enableIvy in tsconfig.json. In case it is false, everything works fine, but if true, auto-generated columns are rendered instead.

I think the problem could be related to this:
https://angular.io/guide/ivy-compatibility-examples#content-children-descendants
so setting {descendants: true} could help (ng-container is probably not considered as direct child)

**To Reproduce**
Example of working example is here: https://stackblitz.com/edit/angular-jy22cj
I am not able to set enableIvy to false in Stackblitz's tsconfig.json, so I attached
[aaa.zip](https://github.com/telerik/kendo-angular/files/4218738/aaa.zip)
 the zip example.
Try changing tsconfig.json, so enableIvy has values true/false and then restart the app.

**Expected behavior**
Columns should be generated from bound property, not auto-generated.

**Screenshots**
OK (enableIvy: false):
![image](https://user-images.githubusercontent.com/39991506/74729480-d3a0b580-5244-11ea-960b-e61d4fb6b935.png)

WRONG (enableIvy: true):
![image](https://user-images.githubusercontent.com/39991506/74729367-a227ea00-5244-11ea-8fe8-e7a8d171cfa9.png)

**Browser**
 - OS: Win 10
 - Browser: Chrome
 - Version: 80.0.3987.106

**Additional context**

**Comment by tsvetomir** [comment]
Seems to be caused by angular/angular#34768, similar to telerik/kendo-angula#2743.

The fix has landed [in master](https://github.com/angular/angular/commit/3f4e02b8c7eaae636feb3b191b73c5918fc015be) and should be released soon.

**Comment by tsvetomir** [comment]
The issue appears to be fixed in Angular 9.0.2

Please note that the `@progress/kendo-resize-sensor` package is no longer in use and should be removed. I'm not sure why yarn is not displaying the deprecation notice.
